plugins/flp_import/CMakeLists.txt

```diff
@@ -3,4 +3,8 @@ INCLUDE(BuildPlugin)
 INCLUDE_DIRECTORIES(unrtf)
 
 ADD_DEFINITIONS(--std=c++0x)
+IF(HOMEBREW)
+	ADD_DEFINITIONS(-stdlib=libc++)
```

Comment:
@Lukas-W what I'm reading is that `libc++` is Clang and `libstdc++` is GNU which explains why they aren't interchangeable.

I'm adding a `LMMS_BUILD_CLANG` variable to make this a bit more straightforward, since the Clang + Linux users will benefit from this change as well.

-Tres